### PR TITLE
add gamedao, zero, 2075

### PR DESCRIPTION
**Pull request by 2075**

### CODE REVIEW

**Comment by 2075** [comment]
:)

**Comment by 2075** [comment]
hey @e-asia thanks for feedback and snapshot :)
yes, i could redo as three separate pull requests. i do not see the big difference, besides extra work closing, forking, etc three times, as the controller adress is the same for the three and besides the three entities being set up there is no code being added. cheers!

**Comment by e-asia** [comment]
Hey @2075 
I found that it could create bugs like having the css gamedeo class in space 2075.
Also it allows to review an invalid space without blocking the other valid spaces.
I let you clean your commit :)
thanks to create spaces on snapshot

**Comment by 2075** [comment]
hey Jeremy, thanks, hope it works out now :)

File: spaces/coin/skin.scss [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
+  .modal .shell {
     background-image: linear-gradient(0deg, #c0392b 0%, #8e44ad 100%);
-}
+  }
 
-.coin .bg-black {
-	background-color: rgba(142,68,173,0.5);
+  .bg-black {
+    background-color: rgba(142,68,173,0.5);
+  }
```
Comment (e-asia) [inline spaces/coin/skin.scss:16]: Use only theme variables. No extra custom styles are allowed.

Comment (2075) [reply spaces/coin/skin.scss:16]: i guess that's not ours but from coin_artist, innit?

**Comment by bonustrack** [comment]
@2075 Your PR include files from others spaces, you most likely not updated the based branch in your PR. Please update your PR to include only your space files.

**Comment by 2075** [comment]
oh dear, possibly we have synced the upstream again and the pr is on our local master.

**Pull request closed without merging**